From 333422fd06ccf69de8052988ec42398cec12ac3f Mon Sep 17 00:00:00 2001
From: Hugues KAMBA MPIANA <hugues.kambampiana@arm.com>
Date: Fri, 8 Aug 2025 00:18:16 +0100
Subject: [PATCH] platform: corstone1000: Add MM communication buffer
 configs to CMake

Two variables are added into the Corstone-1000 platform
CMake file so the MM communication buffer address and
the page count can be configured.

Upstream-Status: Backport [d49a99d6ade219a617741001fb23016d791855ee]
Signed-off-by: Michael Safwat <michael.safwat@arm.com>
Signed-off-by: Harsimran Singh Tungal <harsimransingh.tungal@arm.com>
---
 platform/providers/arm/corstone1000/platform.cmake | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/platform/providers/arm/corstone1000/platform.cmake b/platform/providers/arm/corstone1000/platform.cmake
index 3aeb4922..aeeb8fb4 100644
--- a/platform/providers/arm/corstone1000/platform.cmake
+++ b/platform/providers/arm/corstone1000/platform.cmake
@@ -12,6 +12,10 @@ set(SMM_SP_HEAP_SIZE 80*1024 CACHE STRING "SMM gateway SP heap size")
 set(PLAT_RSE_COMMS_PAYLOAD_MAX_SIZE 0x43C0 CACHE STRING "Size of the RSS_COMMS_PAYLOAD buffer")
 set(COMMS_MHU_MSG_SIZE 0x4500 CACHE STRING "Max message size that can be transfered via MHU")
 
+set(MM_COMM_BUFFER_ADDRESS "0x00000000 0x81FFF000" CACHE STRING "MM Communication buffer start address")
+set(MM_COMM_BUFFER_PAGE_COUNT 0x1 CACHE STRING "MM Communication buffer page count")
+
+
 target_compile_definitions(${TGT} PRIVATE
 	PLAT_RSE_COMMS_PAYLOAD_MAX_SIZE=${PLAT_RSE_COMMS_PAYLOAD_MAX_SIZE}
 	COMMS_MHU_MSG_SIZE=${COMMS_MHU_MSG_SIZE}
-- 
2.43.0

